src/net/core.py

```diff
@@ -77,6 +80,9 @@ def permuter_data_from_json(
         ),
         source=source,
         target_o_bin=target_o_bin,
+        ign_branch_targets=json_prop(obj, "ign_branch_targets", bool),
+        objdump_path=json_prop(obj, "objdump_path", str),
+        objdump_args=json_prop(obj, "objdump_args", str),
```

Comment:
Here it makes sense to default to True / "" / "" for now to preserve compatibility with old clients